api/logs/v1/http.go

```diff
@@ -120,13 +117,19 @@ func NewHandler(read, tail, write *url.URL, opts ...HandlerOption) http.Handler
 				prometheus.Labels{"group": "logsv1", "handler": "query_range"},
 				proxyRead,
 			))
+
+			// Endpoints exposed by the querier and frontend
+			// See https://grafana.com/docs/loki/latest/api/#microservices-mode
+			addDatasourceLogAPIs(r, proxyRead, c)
+
+			// Legacy APIs for Grafana <= 6
+			addLegacyLogAPIS(r, proxyRead, c)
```

Comment:
Please add the handlers inline here. Just like all the other handlers in the file. :)

Reply:
I did this initially, but the linter would fail because the function had too many lines